hs_core/discovery_parser.py

```diff
@@ -111,7 +111,6 @@ class ParseSQ(object):
         'modified',
         'organization',
         'publisher',
```

Comment:
I don't think I'm using discovery_parser.py can it and all related tests and files be deleted or is it tied to the Indexing?

Reply:
@cuahsimarko It's not tied to indexing.

Reply:
@cuahsimarko I can clean up what is used and not used later. Would prefer to do this in a separate PR in order to expedite these particular changes.

Reply:
@alvacouch yes, let's have some of the dust settle on Discover changes for now and revisit removal of stale files later.